types/annote-document/annote-document.ts

```diff
@@ -0,0 +1,13 @@
+export interface AnnoteDocument {
+  id: string;
```

Comment:
I can change this in a future PR but for id's I would make them more descriptive like documentID. when doing joins between 2 tables also, it's easier to choose the right columns as you can do documentID=userID. https://stackoverflow.com/questions/6469730/is-it-better-to-name-the-primary-key-column-id-or-id